app/uk/gov/hmrc/cdsimportsdds/util/MongoFormatters.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsdds.util
+
+import java.time.Instant
+
+import play.api.libs.json.{JsError, JsNumber, JsObject, JsResult, JsSuccess, JsValue, Json, OFormat}
+import uk.gov.hmrc.cdsimportsdds.models.ImportsDeclaration
+
+object MongoFormatters {
+
+  object ImportsDeclaration {
+
+    implicit val formatInstant: OFormat[Instant] = new OFormat[Instant] {
+      override def writes(datetime: Instant): JsObject =
+        Json.obj("$date" -> datetime.toEpochMilli)
+
+      override def reads(json: JsValue): JsResult[Instant] = ???
+    }
+    implicit val format: OFormat[ImportsDeclaration] = Json.format[ImportsDeclaration]
```

Comment:
shouldn't this `format` needs to be defined in companion object of `ImportsDeclaration`? 
If we do that, then we could remove it from `RESTFormatters` and don't need to put the explicit imports for it anywhere (except the `DeclarationRepository`)

Reply:
I want to keep the model free from technology specific code as per the ports and adapters pattern (aka hexagonal architecture). We could move the formatters to their respective ports i.e. the REST ones would live in the controllers inbound port and the Mongo ones would live in the repositories outbound port